specs/DisableCrashReporting.md

```diff
@@ -0,0 +1,120 @@
+Custom Crash Reports
+===
+
+# Background
+Currently, when a WebView2 is created a WebView2 crashpad process will be created in addition to starting a WebView2 browser process. If any WebView2 process crashes, the crashpad process will create crash dumps and send them to Microsoft for diagnosis. This document covers new APIs to allow the end developer to customize crash reporting to help when running diagnostics and doing analysis. They can set the `CoreWebView2EnvironmentOptions.IsCustomCrashReportingEnabled` property to not have crash dumps sent to Microsoft and use the `CoreWebView2Environment.CrashDumpFolderPath` property to locate crash dumps and do customization with them instead.
```

Comment:
Is "crashpad" a publicly documented thing? Or is just a WebView2 internal thing?

Maybe just "The WebView2 platform includes a process that monitors for crashes and sends crash dumps to Microsoft for diagnosis."

Reply:
Its inherited from chromium and it is a public name, but I think your more descriptive name works better regardless.

Reply:
Please update any docs that say crashpad to give a more descriptive description rather than name it.